src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (
+ * see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+  let prop = [];
+  const strToArr = sourceString.split(';').map(key => key.trim());
+
+  for (let i = 0; i < strToArr.length; i++) {
+    prop = strToArr[i].split(':').map(key => key.trim());
```

Comment:
```suggestion
    const [key, value] = strToArr[i].split(':').map(key => key.trim());
```
destructure array instead read by index